postinstall.js

```diff
@@ -1,9 +1,14 @@
 const pkg = require('./package.json');
+const fs = require('fs').promises;
 
 const version = parseFloat(process.version.substr(1));
 const minimum = parseFloat(pkg.engines.node.match(/\d+/g).join('.'));
 
-module.exports = function () {
+module.exports = async function () {
+  // support node 14
```

Comment:
Could you check for the current node major version `process.versions.node.split('.')[0]` and conditionally run a code block only for node 14?

Reply:
It’s necessary for every node version, as running `npm i` mutates the lockfile

Reply:
So the lock file that is currently merged contains a manual change that is overwritten when `npm i` is run?

Reply:
Yes, that is correct

Reply:
Seems like a npm issue, not a Parse Server issue, so maybe we rather instruct developers how to configure git to replace the protocol? 

`sudo git config --system url."https://github".insteadOf "ssh://git@github"`

See https://github.com/parse-community/parse-server/pull/8381 which configures git only for Node 14 in the workflow.